beacon-chain/node/node.go

```diff
@@ -397,17 +397,16 @@ func initSyncWaiter(ctx context.Context, complete chan struct{}) func() error {
 	}
 }
 
-func newRouter(cliCtx *cli.Context) *mux.Router {
+func newRouter(cliCtx *cli.Context) *middleware.NormalizeQueryValuesHandler {
```

Comment:
this doesn't seem good for the long term... we should probably have a function that returns a list of middleware instead of returning a specific one